### PR TITLE
TSDK-345 Simplify deploy of Demo App

### PR DESCRIPTION
**Pull request by mundacho**
## Purpose

The purpose of this PR is to simplify the deploy of the demo application.

## Approach

The new demo application adds a new contract to configure the application. This allows the UI and the trigger to get the data needed (addresses, and the parties IDs)

## Testing

This was tested locally and on DAML Hub.

## Tickets

* closes Topl/Brambl#235

### CODE REVIEW

File: js-daml-pollnft-app/ui/src/components/App.tsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ import LoginScreen from "./LoginScreen";
 import MainScreen from "./MainScreen";
 import { createLedgerContext } from "@daml/react";
 import DamlHub, {
-  damlHubLogout,
+  // damlHubLogout, FIXME
```
Comment (hwbehrens) [inline js-daml-pollnft-app/ui/src/components/App.tsx:9]: What's this about?

File: js-daml-pollnft-app/ui/src/components/MainScreen.tsx [hunk 2]
```diff
@@ -33,16 +32,21 @@ const MainScreen: React.FC = ({ }) => {
   const [walletAddress, setWalletAddress] = useState<undefined | string>(undefined)
   const [currentState, setCurrentState] = useState<StateType>(initialState)
 
-
-  if (currentState == initialState) {
+  if (typeof topl === "undefined") {
+    return appContainer(<>
+        <Alert key='danger' variant='danger'>
+            You need to setup your Ribn Wallet Extension to test this app.
```
Comment (hwbehrens) [inline js-daml-pollnft-app/ui/src/components/MainScreen.tsx:38]: In this context, setup means "have the extension installed"?

**Review by hwbehrens — APPROVED** [review]
Other than the two questions I added, I think it looks good. Thanks!

File: js-daml-pollnft-app/ui/src/components/App.tsx (hunk 1 above)
Comment (mundacho) [reply js-daml-pollnft-app/ui/src/components/App.tsx:9]: yes, that is just an unused dependency... I commented it out and put fixme to be able to go back to it if removing it caused problems. will remove.

File: js-daml-pollnft-app/ui/src/components/MainScreen.tsx (hunk 2 above)
Comment (mundacho) [reply js-daml-pollnft-app/ui/src/components/MainScreen.tsx:38]: yes, will fix, thanks

File: js-daml-pollnft-app/ui/src/components/App.tsx (hunk 1 above)
Comment (mundacho) [reply js-daml-pollnft-app/ui/src/components/App.tsx:9]: It was already removed...

**Pull request merged**